src/main/resources/templates/groups.html

```diff
@@ -15,7 +15,17 @@
     </nav>
 </header>
 <main th:fragment="bodycontent">
-    <!-- Restlichen Inhalt hier einfügen! -->
+    <table>
+        <table>
+            <tr>
+                <th>Name</th>
+                <th>Typ</th>
+            </tr>
+            <tr th:each="group : ${groups}" th:href="@{/dir/${group.id}}">
+                <td th:text="${group.directoryName}">%Name%</td>
```

Comment:
this needs to get the name of the owning group of the directory, as the name of the root directory will be `""` (empty string).